tomni/annotation_manager/annotations/ellipse/main.py

```diff
@@ -51,37 +55,52 @@ def __init__(
         else:
             self.radius_y: float = radius_x
         self.rotation: float = rotation
-
-        # featues
-        all_features = [
-            "area",
-            "circularity",
-            "aspect_ratio",
-            "perimeter",
-        ]
+        self._pixel_density = pixel_density
+        self._metric_unit = metric_unit
+
+        self._area: Union[float, None] = None
+        self._aspect_ratio: Union[float, None] = None
+        self._average_diameter: Union[float, None] = None
+        self._circularity: Union[float, None] = None
+        self._convex_hull_area: Union[float, None] = None
+        self._major_axis: Union[float, None] = None
+        self._minor_axis: Union[float, None] = None
+        self._perimeter: Union[float, None] = None
+        self._roundness: Union[float, None] = None
```

Comment:
```suggestion
        for attr in self._all_features.keys():
            setattr(self, f"_{attr}", None)
```

Reply:
Problem with this method, is that with the American Donut implementation _outer_area is going to have to be initialized however, it will not be part of self._all_features since we dont want this as an output feature.

Reply:
Or initialize that seperately